canonical/demos.haus
# Updated list display of demo pods

## Done

- Switched to cards instead of using a table
- Added search 
- Added status labels to jobs
- Added start/stop buttons to each pod
- Added filters for status

## QA

- Check the demo is running at https://demos-haus-68.demos.haus/
- Test that search is working
- Test that filters are working (independently of search)
- Test that clicking start/stop actually changes the state of a deployment (try starting a failed build e.g. lxd-ui-307-demos-haus)

## Issue / Card

Fixes [WD-7572](https://warthogs.atlassian.net/browse/WD-7572?atlOrigin=eyJpIjoiZTFhZTJjZDBlNDJiNGNlZGExODU1MzU4MGVjNDkwMmYiLCJwIjoiaiJ9)


[WD-7572]: https://warthogs.atlassian.net/browse/WD-7572?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
